packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/done.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Edit2 as EditIcon } from 'react-feather';
+
+import { Price } from '@magento/peregrine';
+
+import { mergeClasses } from '../../../classify';
+import Icon from '../../Icon';
+import LoadingIndicator from '../../LoadingIndicator';
+import defaultClasses from './done.css';
+
+const editIconAttrs = {
+    color: 'black',
+    width: 18
+};
+
+const Done = props => {
+    const {
+        isLoading,
+        selectedShippingMethod,
+        shippingMethods,
+        showEditMode
+    } = props;
+
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    let contents;
+    if (isLoading) {
+        contents = (
+            <LoadingIndicator classes={{ root: classes.loading_root }} />
+        );
+    } else if (!selectedShippingMethod) {
+        // Error state.
+        contents = (
+            <span
+                className={classes.error}
+            >{`Error loading selected shipping method. Please select again.`}</span>
+        );
+    } else {
+        const selectedShippingMethodObject = shippingMethods.find(method => {
+            return selectedShippingMethod === method.serializedValue;
+        });
+        const { amount, method_title } = selectedShippingMethodObject;
+        const { currency, value } = amount;
+
+        const priceElement = value ? (
+            <div>
+                <Price value={value} currencyCode={currency} />
+            </div>
+        ) : (
+            <span className={classes.free}>Free</span>
+        );
+
+        contents = (
+            <div className={classes.contents}>
+                <span>{method_title}</span>
+                {priceElement}
+            </div>
+        );
+    }
+
+    return (
+        <div className={classes.root}>
+            <button className={classes.button} onClick={showEditMode}>
+                <div className={classes.container}>
+                    <span className={classes.title_container}>
```

Comment:
Per the discussion _somewhere_ I think we're leaning toward BEM with camel case ie `titleContainer`. I haven't checked the rest of your css but I would use it going forward.

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2280/commits/8668ca69cc2a4fd82eb6b86ea658b24e0796366d.

The checkout page itself has `snake_case` names but I didn't change that here. All of my styles updated in the commit above.